qa/rpc-tests/wallet_dumpsigma.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import time
+import os
+from decimal import *
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+from test_framework.test_helper import get_dumpwallet_otp
+
+#Scenario:
+#Generate 401 block to activate sigma
+#Mint sigmacoins of all denominations multiple 3
+#generate 6 blocks
+#spend sigma of all denominations
+#generate one block
+#Mint sigma of all denominations and not confirm them
+#getlistunspendsigma
+#getlistspendsigma
+#dumpwallet
+#importwallet
+#verify actual list unspendsigma
+#verify actual list spend sigma
+
+class WalletDumpSigmaTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 4
+        self.setup_clean_chain = False
+
+    def setup_nodes(self):
+        # This test requires mocktime
+        enable_mocktime()
+        return start_nodes(self.num_nodes, self.options.tmpdir)
+
+    def run_test(self):
+        getcontext().prec = 6
+        self.nodes[0].generate(401)
+        self.sync_all()
+
+        sigma_denoms = [0.05, 0.1, 0.5, 1, 10, 25, 100]
+
+        # make confirmed mints and spends
+        denom_sum = sum(sigma_denoms)
+
+        # mint, full confirmation
+        self.nodes[0].mint("{0:.2f}".format(3*denom_sum))
+        self.nodes[0].generate(6)
+
+        # spend, single confirmation
+        self.nodes[0].spendmany("", {'THAYjKnnCsN5xspnEcb1Ztvw4mSPBuwxzU': denom_sum})
+        self.nodes[0].generate(1)
+
+        #make unconfirmed mints and spends
+        #self.nodes[0].mint(denom_sum)
+        #self.nodes[0].spendmany("", {'THAYjKnnCsN5xspnEcb1Ztvw4mSPBuwxzU': denom_sum})
+
+        #get list of unspent mints and spends, mints
+        sigma_mints = self.nodes[0].listsigmamints()
+        sigma_unspentmints = self.nodes[0].listunspentsigmamints(100)
```

Comment:
I think this one will give the empty `sigma_unspentmints` due to we want at least 100 confirmation but the mints we just generated had only 7 confirmations.

Reply:
Correct. I have changed the number of confs to `1`.